app/views/planning_applications/_decision_notice.html.erb

```diff
@@ -41,7 +41,11 @@
     <% end  %>
     <p class="govuk-body">
       <strong>
-        The proposal <%= @planning_application.granted? ? "complies" : "does not comply" %> due to the following reason(s):
+        <% if @planning_application.application_type.name == "planning_permission" %>
+          Permission is <%= @planning_application.granted? ? "granted" : "refused" %> due to the following reason(s):
+        <% else %>
+          The proposal <%= @planning_application.granted? ? "complies" : "does not comply" %> due to the following reason(s):
+        <% end %>
```

Comment:
These could be extracted out into locales in a much cleaner way, e.g.

``` yaml
en
  planning_applications:
    decisions:
      planning_permission:
        granted: Permission is granted due to the following reason(s):
        rejected: Permission is rejected due to the following reason(s): 
      lawful_development: 
        granted: The proposal complies due to the following reasons(s):
        rejected: The proposal does not comply due to the following reasons(s):
```

``` erb
<%= t(@planning_application.decision, scope: :"planning_applications.decisions.#{@planning_application.application_type.name}") %>
```

However it's probably not necessary for now

Reply:
I agree, I think is best to extract them on yml!